crates/sqllib/src/lib.rs

```diff
@@ -22,16 +24,19 @@ pub mod operators;
 pub mod source;
 #[doc(hidden)]
 pub mod string;
+#[doc(hidden)]
```

Comment:
is this enough, or has each function have to be tagged too?

Reply:
looking at cargo doc --open this is enough